edgelet/iotedge/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-#![deny(rust_2018_idioms, warnings)]
-#![deny(clippy::all, clippy::pedantic)]
+// #![deny(rust_2018_idioms, warnings)]
```

Comment:
Revert this.

Reply:
oops, thought I caught that earlier, will do :sweat_smile: